src/OSPSuite.Core/Serialization/Xml/OriginDataCacheXmlSerializer.cs

```diff
@@ -0,0 +1,64 @@
+using System.Xml.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Core.Serialization.Xml
+{
+   public class OriginDataCacheXmlSerializer : OSPSuiteXmlSerializer<OriginDataCache>
+   {
+      // public class CategoryItemXmlSerializer<TItem> : ObjectBaseXmlSerializer<TItem> where TItem : CategoryItem
```

Comment:
Depending what I need to do with CalculationMethodCache, I might need this to serialize.

Reply:
you don't need this